tests/viz_tests.py

```diff
@@ -1062,67 +1013,6 @@ def test_parse_coordinates_raises(self):
         with self.assertRaises(SpatialException):
             test_viz_deckgl.parse_coordinates("fldkjsalkj,fdlaskjfjadlksj")
 
-    @patch("superset.utils.core.uuid.uuid4")
-    def test_filter_nulls(self, mock_uuid4):
-        mock_uuid4.return_value = uuid.UUID("12345678123456781234567812345678")
-        test_form_data = {
-            "latlong_key": {"type": "latlong", "lonCol": "lon", "latCol": "lat"},
-            "delimited_key": {"type": "delimited", "lonlatCol": "lonlat"},
-            "geohash_key": {"type": "geohash", "geohashCol": "geo"},
-        }
-
-        datasource = self.get_datasource_mock()
-        expected_results = {
-            "latlong_key": [
-                {
-                    "clause": "WHERE",
-                    "expressionType": "SIMPLE",
-                    "filterOptionName": "12345678-1234-5678-1234-567812345678",
-                    "comparator": "",
-                    "operator": "IS NOT NULL",
-                    "subject": "lat",
-                    "isExtra": False,
-                },
-                {
-                    "clause": "WHERE",
-                    "expressionType": "SIMPLE",
-                    "filterOptionName": "12345678-1234-5678-1234-567812345678",
-                    "comparator": "",
-                    "operator": "IS NOT NULL",
-                    "subject": "lon",
-                    "isExtra": False,
-                },
-            ],
-            "delimited_key": [
-                {
-                    "clause": "WHERE",
-                    "expressionType": "SIMPLE",
-                    "filterOptionName": "12345678-1234-5678-1234-567812345678",
-                    "comparator": "",
-                    "operator": "IS NOT NULL",
-                    "subject": "lonlat",
-                    "isExtra": False,
-                }
-            ],
-            "geohash_key": [
-                {
-                    "clause": "WHERE",
-                    "expressionType": "SIMPLE",
-                    "filterOptionName": "12345678-1234-5678-1234-567812345678",
-                    "comparator": "",
-                    "operator": "IS NOT NULL",
-                    "subject": "geo",
-                    "isExtra": False,
-                }
-            ],
-        }
-        for mock_key in ["latlong_key", "delimited_key", "geohash_key"]:
-            test_viz_deckgl = viz.BaseDeckGLViz(datasource, test_form_data.copy())
-            test_viz_deckgl.spatial_control_keys = [mock_key]
-            test_viz_deckgl.add_null_filters()
-            adhoc_filters = test_viz_deckgl.form_data["adhoc_filters"]
-            assert expected_results.get(mock_key) == adhoc_filters
-
```

Comment:
Again, the specific `spatial_control_keys` has been removed, hence this test is no longer needed.